index.html

```diff
@@ -53,7 +53,7 @@
 
     <!-- Title -->
 
-    <title>CALCULATOR | YOSHITHA</title>
+    <title><b>CALCULATOR | YOSHITHA</b></title>
```

Comment:
This isn't the one wanted to bold its the main topic "CALCULATOR | YOSHITHA" @Arcturus22